pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "import-linter"
-version = "1.12.0"
+version = "1.12.1"
```

Comment:
Please leave the version number as-is, I'll manage the release in a separate PR.